docs/references/types.md

```diff
@@ -76,11 +76,13 @@ By default, a `strawberry_django` type will get data from the default manager fo
 You can implement a custom `get_queryset` to your type to do some extra processing to the default queryset,
 like filtering it further.
 
+> Warning! The `self` parameter was removed in version: [`0.5.4`](https://github.com/strawberry-graphql/strawberry-graphql-django/releases/tag/v0.5.4). Previously, the method signature looked like: `get_queryset(self, queryset, info):`.
+
 ```python
 @strawberry.django.type(models.Fruit)
 class Berry:
 
-    def get_queryset(self, queryset, info):
```

Comment:
Actually this should be a `classmethod`:

```python
@classmethod
def get_queryset(cls, queryset, info):
    ...
```

The warning should also be updated accordingly

Reply:
Are you saying that the old method `def get_queryset(self, queryset, info):` should be `def get_queryset(cls, queryset, info):`? Or are you talking about the new method `def get_queryset(queryset, info):` that should be changed?

Reply:
The method was actually wrong before. It should always have been `def get_queryset(cls, queryset, info):` decorated with a `@classmethod`, because it is called from the class

Reply:
I just ran into the documentation issue and then traced my way here from discord discussion. 

Would be happy to submit the requested changes in a PR to your branch @fireteam99 or start a new PR here. Let me know if either would be helpful.

Reply:
Sorry for the late response, I'll update the warning accordingly and request a re-review!

Reply:
@bellini666 do the updated docs look correct?

Reply:
Yeap! :)

Reply:
sorry for the delay, i think I've addressed everything